api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<blockquote>Next we will want to give our users friendly wording when they read their anomaly alerts&nbsp;this is optional but for many readers "Water Loss" makes more sense than "Evapotranspiration"<strong>.&nbsp;The </strong>variable data is defined as one series from the land_temperature_array.</blockquote><p> </p><p>Which variable data?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I'm referring the the variable called 'data' so ill put it in quotes</p>